generators/client/files-angular.js

```diff
@@ -461,7 +461,9 @@ const files = {
                 'spec/helpers/_mock-login.service.ts',
                 'spec/helpers/_mock-event-manager.service.ts',
                 'spec/helpers/_mock-active-modal.service.ts',
-                'spec/helpers/_mock-state-storage.service.ts'
+                'spec/helpers/_mock-state-storage.service.ts',
+                'spec/app/shared/model/_request-util.spec.ts',
+                'spec/app/shared/model/_response-wrapper.spec.ts'
```

Comment:
unfortunately this file is being removed in another PR, so you may want to remove the test as well

Reply:
oh, I see, good catch!
would you like me to hold on with this PR and wait for #6951 to be completed, or should I make corrections to adjust to changes in HttpClient upgrade and proceed independently?